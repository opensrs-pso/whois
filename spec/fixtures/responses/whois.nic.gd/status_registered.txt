
domain name: GOOGLE.GD
registrar: MarkMonitor Inc.
url: www.markmonitor.com
status: serverUpdateProhibited
status: serverTransferProhibited
status: serverRenewProhibited
status: serverDeleteProhibited
status: clientupdateprohibited
status: clienttransferprohibited
created date: 2006-12-11 00:00:00
updated date: 2013-03-01 00:02:54
expiration date: 2013-12-11 00:00:00

owner-contact: P-GFI26
owner-organization: Google, Inc.
owner-name: Google, Inc.
owner-street: 1600 Amphitheatre Parkway
owner-city: Mountain View
owner-zip: 94043
owner-country: US
owner-phone: +1.6503300100
owner-fax: +1.6506181499
owner-email: dns-admin@google.com

admin-contact: P-GFI26
admin-organization: Google, Inc.
admin-name: Google, Inc.
admin-street: 1600 Amphitheatre Parkway
admin-city: Mountain View
admin-zip: 94043
admin-country: US
admin-phone: +1.6503300100
admin-fax: +1.6506181499
admin-email: dns-admin@google.com

tech-contact: P-GFI26
tech-organization: Google, Inc.
tech-name: Google, Inc.
tech-street: 1600 Amphitheatre Parkway
tech-city: Mountain View
tech-zip: 94043
tech-country: US
tech-phone: +1.6503300100
tech-fax: +1.6506181499
tech-email: dns-admin@google.com

billing-contact: P-UGM21
billing-organization: MarkMonitor
billing-name: UNKNOWN MarkMonitor
billing-street: 10400 Overland Road
billing-city: Idaho
billing-zip: 83709
billing-country: US
billing-phone: +1.2083895740
billing-fax: +1.2083895771
billing-email: ccops@markmonitor.com

nameserver: ns1.google.com
nameserver: ns2.google.com

; Please register your domains at; www.markmonitor.com

