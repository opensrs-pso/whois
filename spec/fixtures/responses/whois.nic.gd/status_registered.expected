#disclaimer
  should: %s == nil


#domain
  should: %s == "google.gd"

#domain_id
  should: %s raise_error(Whois::AttributeNotSupported)


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2006-12-11 00:00:00")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-03-01 00:02:54")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-12-11 00:00:00")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "MarkMonitor Inc."
  should: %s.organization == nil
  should: %s.url          == "www.markmonitor.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == "P-GFI26"
  should: %s[0].name          == "Google, Inc."
  should: %s[0].organization  == "Google, Inc."
  should: %s[0].address       == "1600 Amphitheatre Parkway"
  should: %s[0].city          == "Mountain View"
  should: %s[0].zip           == "94043"
  should: %s[0].state         == nil
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.6503300100"
  should: %s[0].fax           == "+1.6506181499"
  should: %s[0].email         == "dns-admin@google.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id            == "P-GFI26"
  should: %s[0].name          == "Google, Inc."
  should: %s[0].organization  == "Google, Inc."
  should: %s[0].address       == "1600 Amphitheatre Parkway"
  should: %s[0].city          == "Mountain View"
  should: %s[0].zip           == "94043"
  should: %s[0].state         == nil
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.6503300100"
  should: %s[0].fax           == "+1.6506181499"
  should: %s[0].email         == "dns-admin@google.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id            == "P-GFI26"
  should: %s[0].name          == "Google, Inc."
  should: %s[0].organization  == "Google, Inc."
  should: %s[0].address       == "1600 Amphitheatre Parkway"
  should: %s[0].city          == "Mountain View"
  should: %s[0].zip           == "94043"
  should: %s[0].state         == nil
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.6503300100"
  should: %s[0].fax           == "+1.6506181499"
  should: %s[0].email         == "dns-admin@google.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
